proto/gz/msgs/dvl_beam_state.proto

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+syntax = "proto3";
+package gz.msgs;
+option java_package = "com.gz.msgs";
+option java_outer_classname = "DVLBeamStateProtos";
+
+/// \ingroup gz.msgs
+/// \interface DVLBeamState
+/// \brief State of some acoustic beam in a Doppler Velocity Log sensor. Each beam, when locked,
+/// estimates the distance to the reflecting target as well as its velocity in some reference 
+/// frame. Some additional attributes may also be available.
+
+import "gz/msgs/dvl_kinematic_estimate.proto";
+import "gz/msgs/dvl_range_estimate.proto";
+
+message DVLBeamState
+{
+  /// \brief Beam ID.
+  int32 id = 1;
+
+  /// \brief Beam velocity estimate.
+  DVLKinematicEstimate velocity = 2;
+
+  /// \brief Beam range estimate.
+  DVLRangeEstimate range = 3;
+
+  /// \brief Beam signal strength indicator.
+  double rssi = 4;
+
+  /// \brief Measured background noise spectral density.
+  double nsd = 5;
```

Comment:
Not familiar with this term, does it have units?

Reply:
Added units in https://github.com/gazebosim/gz-msgs/pull/317/commits/d27b90f407fedd0d2e34aabdff7b4be2ebfc70b1. Noise power spectral density characterize how much noise power is there per unit bandwidth.